extensions/2.0/Khronos/KHR_texture_basisu/README.md

```diff
@@ -0,0 +1,153 @@
+
+When the extension is used, it's allowed to use value `image/ktx2` for the `mimeType` property of images that are referenced by the `source` property of `KHR_texture_basisu` texture extension object.
+
+## glTF Schema Updates
+
+The `KHR_texture_basisu` extension is added to the `textures` object and specifies a `source` property that points to the index of the `image` which defines a reference to the KTX2 image with Basis Universal supercompression.
+
+The following glTF will load `image.ktx2` in clients that support this extension, and otherwise fall back to `image.png`.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_basisu"
+    ],
+    "textures": [
+        {
+            "source": 0,
+            "extensions": {
+                "KHR_texture_basisu": {
+                    "source": 1
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "uri": "image.png"
+        },
+        {
+            "uri": "image.ktx2"
+        }
+    ]
+}
+```
+
+When used in the glTF Binary (GLB) format the `image` that points to the KTX2 resource uses the `mimeType` value of `image/ktx2`.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_basisu"
+    ],
+    "textures": [
+        {
+            "source": 0,
+            "extensions": {
+                "KHR_texture_basisu": {
+                    "source": 1
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "mimeType": "image/png",
+            "bufferView": 1
+        },
+        {
+            "mimeType": "image/ktx2",
+            "bufferView": 2
+        }
+    ]
+}
+```
+
+### Using Without a Fallback
+
+To use KTX2 image with Basis Universal supercompression without a fallback, define `KHR_texture_basisu` in both `extensionsUsed` and `extensionsRequired`. The `texture` object will then have its `source` property omitted as shown below.
+
+```json
+{
+    "asset": {
+        "version": "2.0"
+    },
+    "extensionsUsed": [
+        "KHR_texture_basisu"
+    ],
+    "extensionsRequired": [
+        "KHR_texture_basisu"
+    ],
+    "textures": [
+        {
+            "extensions": {
+                "KHR_texture_basisu": {
+                    "source": 0
+                }
+            }
+        }
+    ],
+    "images": [
+        {
+            "uri": "image.ktx2"
+        }
+    ]
+}
+```
+
+### JSON Schema
+
+[texture.KHR_texture_basisu.schema.json](schema/texture.KHR_texture_basisu.schema.json)
+
+## Restrictions on KTX2 Images with Basis Universal Supercompression
+
+When used with glTF 2.0, KTX2 images with Basis Universal supercompression must conform to these additional restrictions:
```

Comment:
While "KTX2 images with Basis Universal supercompression" is conceptually clear, it does imply other required settings. Should we list any of these here?

- supercompressionScheme = 1
- vkFormat = VK_FORMAT_UNDEFINED
- typeSize = 1

Are key/value data and DFD blocks (other than Khronos Basic Data Format Descriptor Block) allowed?

Reply:
`supercompressionScheme`, `vkFormat`, and `typeSize` for BasisU are enforced by the KTX2 spec.

Custom metadata and DFD blocks should be ignored as it's a `KHR` extension.

Reply:
I think it's worth explicitly saying that other methods of supercompression are not allowed by this extension – the other details are (I agree) clear enough in the KTX2 spec.

Reply:
Done.